### PR TITLE
[AJUN-283] PlayerConfigStats not updateing locks

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

Updates logic so affiliators locks represent its status in AAA

## Type of changes

- [ ] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [ ] `feat`: Changes to add a new feature
- [ ] `fix`: Changes to fix a bug
- [x] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [ ] Tests for the changes have been added
- [ ] Necessary documentation is added (if appropriate)
- [ ] Formatted with `cargo fmt --all`
- [ ] Linted with `cargo clippy --all-features --all-targets`
- [ ] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

File: pallets/ajuna-awesome-avatars/src/lib.rs [hunk 1]
```diff
@@ -1290,6 +1285,9 @@ pub mod pallet {
 						affiliate_config.affiliator_enable_fee,
```
Comment (darkfriend77) [inline pallets/ajuna-awesome-avatars/src/lib.rs:1285]: missing the check if it is already an Affiliator we don't want to let them pay two times

File: pallets/ajuna-awesome-avatars/src/lib.rs
Comment (darkfriend77) [inline pallets/ajuna-awesome-avatars/src/lib.rs:1]: enable_set_avatar_price also misses the check if it is already unlocked, on the payed version.

Comment (darkfriend77) [inline pallets/ajuna-awesome-avatars/src/lib.rs:1]: enable_avatar_transfer also misses the check if it is already unlocked, on the payed version.

**Review by darkfriend77 — CHANGES_REQUESTED** [review]

File: pallets/ajuna-awesome-avatars/src/lib.rs (hunk 1 above)
Comment (DidacSF) [reply pallets/ajuna-awesome-avatars/src/lib.rs]: Added checks!

File: pallets/ajuna-awesome-avatars/src/lib.rs
Comment (DidacSF) [reply pallets/ajuna-awesome-avatars/src/lib.rs:1]: Added checks!

**Pull request merged**